static/src/javascripts/projects/common/modules/experiments/tests/loyal-adblocking-survey.js

```diff
@@ -0,0 +1,63 @@
+define([
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/storage',
+    'common/modules/commercial/user-features',
+    'common/modules/commercial/survey/survey-simple'
+], function (
+    $,
+    config,
+    detect,
+    storage,
+    userFeatures,
+    SurveySimple
+) {
+    function getSurveyLink(adblockUsed) {
+        var ophanId = '&ophanId=' + config.ophan.pageViewId;
+        return adblockUsed ? 'https://surveys.theguardian.com/R.aspx?a=684&as=AC5tF8XE6G&t=1' + ophanId : 'https://surveys.theguardian.com/R.aspx?a=685&as=Aj5jo96J9j&t=1' + ophanId;
+    }
+
+    return function () {
+        this.id = 'LoyalAdblockingSurvey';
+        this.start = '2016-03-21';
+        this.expiry = '2016-05-31';
+        this.author = 'Zofia Korcz';
+        this.description = 'An adblock ongoing survey for all loyal users';
+        this.audience = 0;
```

Comment:
Not yet. First I wanted to set up an AB test with no audience, so we can test it by opting in manually.